android/app/src/googleplay/res/layout/fragment_map.xml

```diff
@@ -1,5 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.google.android.gms.maps.MapView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/map"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
RelativeLayout with single child does not make sense. Make FrameLayout root layout